src/contract.rs

```diff
@@ -238,10 +238,40 @@ fn process_bulk_order_placements(
 }
 
 fn process_bulk_order_cancellations(
-    _: DepsMut<SeiQueryWrapper>,
-    _: Vec<u64>,
+    deps: DepsMut<SeiQueryWrapper>,
+    ids_to_cancel: Vec<u64>,
 ) -> Result<Response, ContractError> {
-    // TODO
+    for id in ids_to_cancel {
+        let order_result = get_order(deps.storage, id);
+        if let Ok(order) = order_result {
+            delete_order(deps.storage, id);
```

Comment:
should we delete order at the end of the loop (line 270), in case there is any panic happening from the save_balance? in that case the state will still be uncorrupted

Reply:
sei chain would ensure that any panic being rolled back, but i updated it anyway